desktop-widgets/mapwidget.cpp

```diff
@@ -94,7 +93,15 @@ void MapWidget::selectionChanged()
 void MapWidget::selectedDivesChanged(const QList<int> &list)
 {
 	CHECK_IS_READY_RETURN_VOID();
-	MainWindow::instance()->diveList->selectDives(list);
```

Comment:
this was the result of the marble -> qml map port.
at some point we had a compiler macro to smooth switch between the two.